Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -329,19 +348,13 @@ def _createDiagram(self):
                 area.setText("{0}".format(len(area_items)))
 
             label = disjoint_set_label(i, n, simplify=False)
-            head = "<h4>|{}| = {}</h4>".format(label, len(area_items))
-            if len(area_items) > 32:
-                items_str = ", ".join(map(escape, area_items[:32]))
-                hidden = len(area_items) - 32
-                tooltip = ("{}<span>{}, ...</br>({} items not shown)<span>"
-                           .format(head, items_str, hidden))
-            elif area_items:
-                tooltip = "{}<span>{}</span>".format(
-                    head,
-                    ", ".join(map(escape, area_items))
-                )
-            else:
-                tooltip = head
+            tooltip = "<h4>|{}| = {}</h4>".format(label, len(area_items))
+            if self._uses_feature() or not self.rowwise:
+                # Nothing readable to show when matching by identity or equality
```

Comment:
This condition prevents showing hashes on tooltip. Even before this PR, matching by identity showed ids, which also mean nothing to the user.

We could show entire rows instead, but let the user connect the Table widget for this instead.